providers/stores.inc

```diff
@@ -12,3 +12,6 @@
 #endif
 
 STORE("file", "yes", ossl_file_store_functions)
+#ifndef OPENSSL_NO_WINSTORE
+STORE("winstore", "no", ossl_winstore_store_functions)
```

Comment:
The "no" means fips=no - imo there is no reason to have it so as there is no difference between file and winstore stores in this regard.